workspace/assets/js/utils/keys.js

```diff
@@ -0,0 +1,143 @@
+/**
+ * Friendly maping of "all" the keyboard keys
+ * @author Deux Huit Huit
+ */
+(function ($) {
+
+	'use strict';
+
+	App.device.keys = {};
+
+	App.device.keys.code = {
```

Comment:
Please use `$.extends` like in the code. Set the first parameter to be true (Deep copy)